lib/ngtcp2_conn.c

```diff
@@ -1212,7 +1212,6 @@ static ssize_t conn_write_handshake_pkt(ngtcp2_conn *conn, uint8_t *dest,
     }
 
     *pfrc = frc;
-    pfrc = &frc->next;
```

Comment:
I might not be clear, I leave this as is.  Because it is very easy to forget to add this when someone adds new chain after this block later.

Reply:
Done (remove the patch..)